code/game/machinery/doors/door.dm

```diff
@@ -97,6 +98,9 @@
 			return TRUE
 	return FALSE
 
+/obj/machinery/door/proc/try_remove_seal(mob/user)
```

Comment:
requires dm documentation

Reply:
I think I documented the proc correctly in the airlock.dm one, should it be here instead?